mlrun/db/httpdb.py

```diff
@@ -1812,6 +1814,8 @@ def create_feature_vector(
         """
         if isinstance(feature_vector, schemas.FeatureVector):
             feature_vector = feature_vector.dict()
+        elif isinstance(feature_vector, FeatureVector):
+            feature_vector = feature_vector.to_dict()
```

Comment:
This change is meaning less unless you change the type-hints to allow it like you did in the feature-set method - `feature_vector: Union[dict, schemas.FeatureVector, FeatureVector]`.
Also, if you change `create_feature_vector` might as well do the same for `store_feature_vector`.